portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,21 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.common.collect.ImmutableList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns list of greeting messages. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final ImmutableList HELLO_MESSAGES = ImmutableList.of("Hello Megan!", "Hello there!", "Hello!");
```

Comment:
Does this compile? I think you might be missing a \<String\> in the definition of ImmutableList (should be private static final ImmutableList\<String\> HELLO_MESSAGES...)

Sorry, I think I forgot to format the brackets in my last comment so they didn't show up :(